core/src/main/antlr/bio/terra/pearl/core/antlr/CohortRule.g4

```diff
@@ -4,8 +4,7 @@ grammar CohortRule;
    See https://github.com/surveyjs/survey-library/blob/master/src/expressions/grammar.pegjs */
 
 // Parser rules
-expr: PAR_OPEN expr PAR_CLOSE | term OPERATOR term | expr AND expr | expr OR expr | NOT expr;
-term: NUMBER | STRING | VARIABLE | BOOLEAN | NULL | FUNCTION_NAME PAR_OPEN term (',' term)* PAR_CLOSE;
+expr: PAR_OPEN expr PAR_CLOSE | term OPERATOR term | expr AND expr | expr OR expr | NOT expr | INCLUDE PAR_OPEN term PAR_CLOSE;term: NUMBER | STRING | VARIABLE | BOOLEAN | NULL | FUNCTION_NAME PAR_OPEN term (',' term)* PAR_CLOSE;
```

Comment:
why did these get put on a single line?

Reply:
I have no idea - I copied and pasted them so I guess copy/paste error. Good catch!